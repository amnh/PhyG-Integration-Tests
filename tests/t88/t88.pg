--build(distance, rdwag, block), inputting fasta files with missing and chopped data
set(seed:1634561640)
set(outgroup:"Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184")
read(fasta:"dec258_12Saln_missing.fas")
read(fasta:"dec258_16Saln_chopped.fasta")
read(include:"Include_dec.txt")
build(distance, rdWag, block, eun, displaytrees:3)
report("t88.csv", data, overwrite)
report("t88.tre", newick, overwrite)