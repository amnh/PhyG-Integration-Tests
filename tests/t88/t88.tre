(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Aristeidae_Aristeomorpha_foliacea_NC_039153:549.5,((Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Cambaridae_Procambarus_clarkii_NC_016926:257.5,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_pallipes_NC_026560:231.5,Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_torrentium_NC_033504:217.5)HTU42:105.5)HTU44:203.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Bythograeoidea_Bythograeidae_Austinograea_rodriguezensis_NC_020312:419.0,(Eucarida_Decapoda_Pleocyemata_Axiidea_Callianassidae_Nihonotrypaea_thermophila_NC_019610:362.5,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Luciferidae_Belzebub_intermedius_NC_039641:395.5,((Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Pleoticus_muelleri_NC_039964:251.5,Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Solenocera_crassicornis_NC_030280:250.0)HTU35:157.5,((((Eucarida_Decapoda_Pleocyemata_Caridea_Nematocarcinoidea_Rhynchocinetidae_Rhynchocinetes_durbanensis_NC_029372:392.5,(((Eucarida_Decapoda_Pleocyemata_Anomura_Galatheoidea_Porcellanidae_Petrolisthes_haswelli_NC_025572:363.5,Eucarida_Decapoda_Pleocyemata_Caridea_Bresilioidea_Alvinocarididae_Opaepele_loihi_NC_020311:351.5)HTU36:213.5,(Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_argus_NC_039671:343.5,Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_cygnus_NC_028024:318.5)HTU40:229.5)HTU32:159.0,(Eucarida_Decapoda_Pleocyemata_Anomura_Paguroidea_Diogenidae_Clibanarius_infraspinatus_NC_025776:366.5,Eucarida_Decapoda_Pleocyemata_Caridea_Atyoidea_Atyidae_Caridina_gracilipes_NC_024751:364.0)HTU38:202.5)HTU37:185.0)HTU34:168.5,(Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_laevis_KX343004:239.0,Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_validus_NC_041107:210.5)HTU45:218.5)HTU33:162.0,((Eucarida_Decapoda_Pleocyemata_Caridea_Alpheoidea_Alpheidae_Alpheus_distinguendus_NC_014883:432.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Dromiacea_Homoloidea_Homolidae_Homologenus_malayensis_NC_026080:370.5,Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Portunoidea_Portunidae_Portunus_trituberculatus_NC_005037:378.5)HTU54:165.0)HTU55:184.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Crangonoidea_Crangonidae_Crangon_hakodatei_KU641481:361.0,(Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184:346.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Potamoidea_Potamidae_Potamiscus_motuoensis_KY285013:359.5,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Spongicola_levigatus_KU188325:394.0)HTU49:192.5)HTU48:168.5)HTU60:172.0)HTU47:158.0)HTU43:162.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Palaemonoidea_Palaemonidae_Hymenocera_picta_NC_039631:365.0,((Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Austinogebia_edulis_NC_019606:315.0,Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Gebiacantha_plantae_NC_041152:351.5)HTU39:145.5,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Sergestidae_Sergia_lucens_NC_037043:361.0,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Globospongicola_spinulatus_KU188326:321.5,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Stenopodidae_Stenopus_hispidus_NC_018097:301.5)HTU50:294.0)HTU53:177.5)HTU51:144.0)HTU52:156.0)HTU41:162.0)HTU46:184.0)HTU56:155.0)HTU57:133.0)HTU58:170.5)HTU59:510.0)HTU31[12216.0];
(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Aristeidae_Aristeomorpha_foliacea_NC_039153:549.5,((Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Cambaridae_Procambarus_clarkii_NC_016926:257.5,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_pallipes_NC_026560:231.5,Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_torrentium_NC_033504:217.5)HTU42:105.5)HTU44:203.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Bythograeoidea_Bythograeidae_Austinograea_rodriguezensis_NC_020312:419.0,(Eucarida_Decapoda_Pleocyemata_Axiidea_Callianassidae_Nihonotrypaea_thermophila_NC_019610:362.5,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Luciferidae_Belzebub_intermedius_NC_039641:395.5,((Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Pleoticus_muelleri_NC_039964:251.5,Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Solenocera_crassicornis_NC_030280:250.0)HTU35:157.5,((((Eucarida_Decapoda_Pleocyemata_Caridea_Nematocarcinoidea_Rhynchocinetidae_Rhynchocinetes_durbanensis_NC_029372:392.5,(((Eucarida_Decapoda_Pleocyemata_Anomura_Galatheoidea_Porcellanidae_Petrolisthes_haswelli_NC_025572:363.5,Eucarida_Decapoda_Pleocyemata_Caridea_Bresilioidea_Alvinocarididae_Opaepele_loihi_NC_020311:351.5)HTU36:213.5,(Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_argus_NC_039671:343.5,Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_cygnus_NC_028024:318.5)HTU40:229.5)HTU32:159.0,(Eucarida_Decapoda_Pleocyemata_Anomura_Paguroidea_Diogenidae_Clibanarius_infraspinatus_NC_025776:366.5,Eucarida_Decapoda_Pleocyemata_Caridea_Atyoidea_Atyidae_Caridina_gracilipes_NC_024751:364.0)HTU38:202.5)HTU37:185.0)HTU34:168.5,(Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_laevis_KX343004:239.0,Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_validus_NC_041107:210.5)HTU45:218.5)HTU33:162.0,((Eucarida_Decapoda_Pleocyemata_Caridea_Alpheoidea_Alpheidae_Alpheus_distinguendus_NC_014883:432.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Dromiacea_Homoloidea_Homolidae_Homologenus_malayensis_NC_026080:370.5,Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Portunoidea_Portunidae_Portunus_trituberculatus_NC_005037:378.5)HTU54:165.0)HTU55:184.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Crangonoidea_Crangonidae_Crangon_hakodatei_KU641481:361.0,(Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184:346.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Potamoidea_Potamidae_Potamiscus_motuoensis_KY285013:359.5,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Spongicola_levigatus_KU188325:394.0)HTU49:192.5)HTU48:168.5)HTU60:172.0)HTU47:158.0)HTU43:162.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Palaemonoidea_Palaemonidae_Hymenocera_picta_NC_039631:365.0,((Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Austinogebia_edulis_NC_019606:315.0,Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Gebiacantha_plantae_NC_041152:351.5)HTU39:145.5,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Sergestidae_Sergia_lucens_NC_037043:361.0,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Globospongicola_spinulatus_KU188326:321.5,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Stenopodidae_Stenopus_hispidus_NC_018097:301.5)HTU50:294.0)HTU53:177.5)HTU51:144.0)HTU52:156.0)HTU41:162.0)HTU46:184.0)HTU56:155.0)HTU57:133.0)HTU58:170.5)HTU59:510.0)HTU31[12216.0];
(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Aristeidae_Aristeomorpha_foliacea_NC_039153:575.5,((Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Cambaridae_Procambarus_clarkii_NC_016926:269.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_pallipes_NC_026560:246.5,Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_torrentium_NC_033504:231.5)HTU42:97.5)HTU44:201.5,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Bythograeoidea_Bythograeidae_Austinograea_rodriguezensis_NC_020312:435.5,(Eucarida_Decapoda_Pleocyemata_Axiidea_Callianassidae_Nihonotrypaea_thermophila_NC_019610:377.5,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Luciferidae_Belzebub_intermedius_NC_039641:403.0,((Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Pleoticus_muelleri_NC_039964:260.5,Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Solenocera_crassicornis_NC_030280:266.0)HTU35:163.0,((((Eucarida_Decapoda_Pleocyemata_Caridea_Nematocarcinoidea_Rhynchocinetidae_Rhynchocinetes_durbanensis_NC_029372:403.5,(((Eucarida_Decapoda_Pleocyemata_Anomura_Galatheoidea_Porcellanidae_Petrolisthes_haswelli_NC_025572:342.0,Eucarida_Decapoda_Pleocyemata_Caridea_Bresilioidea_Alvinocarididae_Opaepele_loihi_NC_020311:374.5)HTU36:182.0,(Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_argus_NC_039671:340.5,Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_cygnus_NC_028024:293.0)HTU40:229.0)HTU32:174.5,(Eucarida_Decapoda_Pleocyemata_Anomura_Paguroidea_Diogenidae_Clibanarius_infraspinatus_NC_025776:362.0,Eucarida_Decapoda_Pleocyemata_Caridea_Atyoidea_Atyidae_Caridina_gracilipes_NC_024751:362.5)HTU38:185.0)HTU37:191.5)HTU34:166.0,(Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_laevis_KX343004:242.0,Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_validus_NC_041107:214.5)HTU45:234.0)HTU33:162.5,((Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184:347.5,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Potamoidea_Potamidae_Potamiscus_motuoensis_KY285013:375.5,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Spongicola_levigatus_KU188325:410.0)HTU49:186.5)HTU48:163.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Alpheoidea_Alpheidae_Alpheus_distinguendus_NC_014883:456.5,(Eucarida_Decapoda_Pleocyemata_Brachyura_Dromiacea_Homoloidea_Homolidae_Homologenus_malayensis_NC_026080:383.5,Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Portunoidea_Portunidae_Portunus_trituberculatus_NC_005037:381.5)HTU55:148.5)HTU56:179.0)HTU47:149.0)HTU43:151.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Palaemonoidea_Palaemonidae_Hymenocera_picta_NC_039631:374.5,((Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Austinogebia_edulis_NC_019606:324.0,Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Gebiacantha_plantae_NC_041152:365.5)HTU39:137.0,((Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Globospongicola_spinulatus_KU188326:329.0,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Stenopodidae_Stenopus_hispidus_NC_018097:306.5)HTU50:282.0,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Sergestidae_Sergia_lucens_NC_037043:388.0,Eucarida_Decapoda_Pleocyemata_Caridea_Crangonoidea_Crangonidae_Crangon_hakodatei_KU641481:428.0)HTU54:154.5)HTU53:151.5)HTU51:150.0)HTU52:163.0)HTU41:154.0)HTU46:188.5)HTU57:153.5)HTU58:138.0)HTU59:168.5)HTU60:520.5)HTU31[12199.0];
