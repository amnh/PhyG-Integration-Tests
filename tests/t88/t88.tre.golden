(Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184:578.5,((Eucarida_Decapoda_Pleocyemata_Caridea_Atyoidea_Atyidae_Caridina_gracilipes_NC_024751:414.5,((Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Portunoidea_Portunidae_Portunus_trituberculatus_NC_005037:328.5,(Eucarida_Decapoda_Pleocyemata_Brachyura_Dromiacea_Homoloidea_Homolidae_Homologenus_malayensis_NC_026080:368.5,Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Bythograeoidea_Bythograeidae_Austinograea_rodriguezensis_NC_020312:334.5)HTU46:154.0)HTU45:162.5,(((Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_argus_NC_039671:336.0,Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_cygnus_NC_028024:330.5)HTU35:225.0,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Sergestidae_Sergia_lucens_NC_037043:300.5,(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Aristeidae_Aristeomorpha_foliacea_NC_039153:295.5,(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Pleoticus_muelleri_NC_039964:258.5,Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Solenocera_crassicornis_NC_030280:268.0)HTU33:102.0)HTU32:132.0)HTU36:192.5)HTU41:160.0,((Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Austinogebia_edulis_NC_019606:271.0,Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Gebiacantha_plantae_NC_041152:315.0)HTU34:151.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Alpheoidea_Alpheidae_Alpheus_distinguendus_NC_014883:418.5,Eucarida_Decapoda_Pleocyemata_Caridea_Crangonoidea_Crangonidae_Crangon_hakodatei_KU641481:313.5)HTU43:261.5)HTU42:187.0)HTU48:164.0)HTU44:151.0)HTU59:144.0,((Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Spongicola_levigatus_KU188325:375.0,((Eucarida_Decapoda_Pleocyemata_Caridea_Palaemonoidea_Palaemonidae_Hymenocera_picta_NC_039631:370.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Nematocarcinoidea_Rhynchocinetidae_Rhynchocinetes_durbanensis_NC_029372:404.5,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Cambaridae_Procambarus_clarkii_NC_016926:272.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_pallipes_NC_026560:257.5,Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_torrentium_NC_033504:238.5)HTU37:109.0)HTU38:200.0)HTU55:157.5)HTU51:179.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Potamoidea_Potamidae_Potamiscus_motuoensis_KY285013:328.0,(Eucarida_Decapoda_Pleocyemata_Anomura_Paguroidea_Diogenidae_Clibanarius_infraspinatus_NC_025776:414.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Bresilioidea_Alvinocarididae_Opaepele_loihi_NC_020311:395.5,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Globospongicola_spinulatus_KU188326:318.0,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Stenopodidae_Stenopus_hispidus_NC_018097:311.0)HTU47:182.0)HTU52:247.5)HTU49:151.0)HTU54:196.5)HTU53:134.5)HTU50:198.0,((Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_laevis_KX343004:268.0,Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_validus_NC_041107:226.0)HTU39:224.5,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Luciferidae_Belzebub_intermedius_NC_039641:394.0,(Eucarida_Decapoda_Pleocyemata_Anomura_Galatheoidea_Porcellanidae_Petrolisthes_haswelli_NC_025572:388.5,Eucarida_Decapoda_Pleocyemata_Axiidea_Callianassidae_Nihonotrypaea_thermophila_NC_019610:358.0)HTU57:178.5)HTU58:193.0)HTU56:162.5)HTU60:132.5)HTU40:482.5)HTU31[11868.0];
(Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184:576.5,(((((Eucarida_Decapoda_Pleocyemata_Anomura_Paguroidea_Diogenidae_Clibanarius_infraspinatus_NC_025776:357.0,Eucarida_Decapoda_Pleocyemata_Caridea_Atyoidea_Atyidae_Caridina_gracilipes_NC_024751:354.0)HTU35:227.0,(Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_argus_NC_039671:317.0,Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_cygnus_NC_028024:323.0)HTU37:237.5)HTU34:210.5,(Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_laevis_KX343004:226.0,Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_validus_NC_041107:186.5)HTU42:234.5)HTU40:185.5,((Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Portunoidea_Portunidae_Portunus_trituberculatus_NC_005037:324.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Dromiacea_Homoloidea_Homolidae_Homologenus_malayensis_NC_026080:354.0,Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Bythograeoidea_Bythograeidae_Austinograea_rodriguezensis_NC_020312:328.0)HTU47:151.0)HTU46:180.5,((Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Sergestidae_Sergia_lucens_NC_037043:262.0,(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Aristeidae_Aristeomorpha_foliacea_NC_039153:264.0,(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Pleoticus_muelleri_NC_039964:225.5,Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Solenocera_crassicornis_NC_030280:237.0)HTU33:108.5)HTU32:151.0)HTU38:171.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Alpheoidea_Alpheidae_Alpheus_distinguendus_NC_014883:470.5,(Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Austinogebia_edulis_NC_019606:244.0,Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Gebiacantha_plantae_NC_041152:289.0)HTU36:208.0)HTU44:177.0)HTU49:139.5)HTU45:148.5)HTU59:158.5,((Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Spongicola_levigatus_KU188325:353.0,((Eucarida_Decapoda_Pleocyemata_Caridea_Palaemonoidea_Palaemonidae_Hymenocera_picta_NC_039631:363.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Nematocarcinoidea_Rhynchocinetidae_Rhynchocinetes_durbanensis_NC_029372:406.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Cambaridae_Procambarus_clarkii_NC_016926:273.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_pallipes_NC_026560:256.5,Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_torrentium_NC_033504:230.5)HTU39:92.5)HTU41:205.5)HTU56:157.0)HTU52:162.5,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Potamoidea_Potamidae_Potamiscus_motuoensis_KY285013:367.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Bresilioidea_Alvinocarididae_Opaepele_loihi_NC_020311:378.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Crangonoidea_Crangonidae_Crangon_hakodatei_KU641481:406.5,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Globospongicola_spinulatus_KU188326:345.0,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Stenopodidae_Stenopus_hispidus_NC_018097:345.0)HTU48:173.5)HTU51:171.0)HTU53:179.0)HTU55:154.5)HTU54:174.5)HTU50:172.0,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Luciferidae_Belzebub_intermedius_NC_039641:392.0,(Eucarida_Decapoda_Pleocyemata_Anomura_Galatheoidea_Porcellanidae_Petrolisthes_haswelli_NC_025572:457.0,Eucarida_Decapoda_Pleocyemata_Axiidea_Callianassidae_Nihonotrypaea_thermophila_NC_019610:364.5)HTU57:147.5)HTU58:207.0)HTU60:153.5)HTU43:497.5)HTU31[11823.0];
(Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184:565.5,((((Eucarida_Decapoda_Pleocyemata_Anomura_Paguroidea_Diogenidae_Clibanarius_infraspinatus_NC_025776:343.5,Eucarida_Decapoda_Pleocyemata_Caridea_Atyoidea_Atyidae_Caridina_gracilipes_NC_024751:357.0)HTU35:224.0,(Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_argus_NC_039671:324.0,Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_cygnus_NC_028024:306.5)HTU37:245.0)HTU34:222.5,((Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Portunoidea_Portunidae_Portunus_trituberculatus_NC_005037:326.5,(Eucarida_Decapoda_Pleocyemata_Brachyura_Dromiacea_Homoloidea_Homolidae_Homologenus_malayensis_NC_026080:299.5,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Bythograeoidea_Bythograeidae_Austinograea_rodriguezensis_NC_020312:336.5,Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Potamoidea_Potamidae_Potamiscus_motuoensis_KY285013:309.0)HTU49:155.5)HTU47:212.5)HTU46:162.0,((Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Sergestidae_Sergia_lucens_NC_037043:265.0,(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Aristeidae_Aristeomorpha_foliacea_NC_039153:261.0,(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Pleoticus_muelleri_NC_039964:233.5,Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Solenocera_crassicornis_NC_030280:249.5)HTU33:165.0)HTU32:138.0)HTU38:211.5,((Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Austinogebia_edulis_NC_019606:274.5,Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Gebiacantha_plantae_NC_041152:301.5)HTU36:188.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Alpheoidea_Alpheidae_Alpheus_distinguendus_NC_014883:397.0,Eucarida_Decapoda_Pleocyemata_Caridea_Crangonoidea_Crangonidae_Crangon_hakodatei_KU641481:332.5)HTU44:239.5)HTU43:175.5)HTU50:154.0)HTU45:155.5)HTU59:149.5,((Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Spongicola_levigatus_KU188325:360.5,((Eucarida_Decapoda_Pleocyemata_Caridea_Palaemonoidea_Palaemonidae_Hymenocera_picta_NC_039631:365.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Nematocarcinoidea_Rhynchocinetidae_Rhynchocinetes_durbanensis_NC_029372:400.5,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Cambaridae_Procambarus_clarkii_NC_016926:269.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_pallipes_NC_026560:251.5,Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_torrentium_NC_033504:228.5)HTU39:102.5)HTU40:207.5)HTU55:156.5)HTU52:166.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Bresilioidea_Alvinocarididae_Opaepele_loihi_NC_020311:364.5,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Globospongicola_spinulatus_KU188326:310.0,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Stenopodidae_Stenopus_hispidus_NC_018097:299.5)HTU48:177.0)HTU53:131.5)HTU54:168.5)HTU51:176.5,((Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_laevis_KX343004:253.5,Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_validus_NC_041107:205.5)HTU41:214.0,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Luciferidae_Belzebub_intermedius_NC_039641:384.5,(Eucarida_Decapoda_Pleocyemata_Anomura_Galatheoidea_Porcellanidae_Petrolisthes_haswelli_NC_025572:370.5,Eucarida_Decapoda_Pleocyemata_Axiidea_Callianassidae_Nihonotrypaea_thermophila_NC_019610:354.5)HTU57:178.5)HTU58:205.0)HTU56:183.0)HTU60:187.0)HTU42:515.5)HTU31[11737.0];
